templates/aws-stack.yml

```diff
@@ -1054,6 +1054,8 @@ Resources:
         - OldestLaunchConfiguration
         - ClosestToNextInstanceHour
       MaxInstanceLifetime: !If [UseMaxInstanceLifetime, !Ref MaxInstanceLifetime, !Ref "AWS::NoValue"]
+      NewInstancesProtectedFromScaleIn: true
```

Comment:
@nitrocode I suspect (but haven’t tested) adding this by merging master will break the `MaxInstanceLifetime` functionality because none of the instances are eligible for ASG initiated termination. I’m not sure if you or anyone else are running stacks from this branch’s template but wanted to call it out in case you are 😄